files/help/ares-log.help

```diff
@@ -0,0 +1,97 @@
+{
+    "display" : ["name", "synopsis", "options", "description", "examples"],
+    "notab":["examples"],
+    "name" : "ares-log - Show and save logs of webOS's apps and services",
+    "synopsis" : {
+        "default" : [
+           "ares-log [OPTION...]",
+            "ares-log [OPTION...] -f, --follow",
```

Comment:
There is no ares-log -cd option.

Reply:
-cd option will be opened when pmlogd is supported. 
I will remove example of -cd option.